examples/support/argo/assets/argo_cm.yaml

```diff
@@ -22,17 +22,17 @@ data:
     end
     return hs
 
-  resource.customizations.health.lifecycle.keptn.sh_KeptnWorkloadInstance: |
```

Comment:
we should update also the repo upstream of Argo 🤔

Reply:
good catch, I checked and argo is still using v1alpha2.KeptnWorkloadInstance, I will try to create a PR to include v1alpha4.KeptnWorkloadVersion

Reply:
Thanks a lot 🙌

Reply:
according to the process in ArgoCD, I created an enhancement proposal https://github.com/argoproj/argo-cd/issues/15922. If this will be accepted, I can follow-up with a contribution to ArgoCD

Reply:
does this mean argo will not show ❤️  with our new version? :(

Reply:
I hope that the proposal will be accepted and then we can follow-up with a PR to ArgoCD

Reply:
You can go ahead and propose a PR directly :) they will appreciate that :) See the past PR from @thschue: https://github.com/argoproj/argo-cd/pull/11716

Reply:
> does this mean argo will not show ❤️ with our new version? :(

not our of the box, but we can apply this CM to it so it can display 💔 / ❤️

Reply:
@thisthat sure, I was just refering to the contributing guide, where they suggest to create a proposal first https://argo-cd.readthedocs.io/en/latest/developer-guide/code-contributions/

But sure, the changes I have prepared locally already, so submitting a PR should take just a while

Reply:
the PR https://github.com/argoproj/argo-cd/pull/15951